lib/form/paragraph.flow

```diff
@@ -595,8 +595,8 @@ breakTextFragment(
 			if (isBiDiEnabled()) {
 				if (i >= ^chrCatPos) chrCatPos := remapMinusOne(getCharCatSplitPos(text, i+1, textLen)-1);
```

Comment:
@brawaga , take a look please. in the getCharCatSplitPos we are looking for some "breaks" so may be it should be generalized